### PR TITLE
implement user and privileges module

### PR DESCRIPTION
**Pull request by m-kharbat**
screen shot 2017-08-22 at 15 55 59
screen shot 2017-08-22 at 15 55 52
screen shot 2017-08-22 at 15 55 42

### CODE REVIEW

File: app/plugins/privileges/privileges.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.user-list__element {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
```
Comment (Autophagy) [inline app/plugins/privileges/privileges.css]: Inconsistent indentation

File: app/plugins/privileges/privileges.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    background-color: #242424;
+    outline:none;
+}
+.table-search__wrapper{
+    position: relative;
+    max-width: 300px;
+}
+
+.cr-clear-privileges-search {
+  float: right;
```
Comment (Autophagy) [inline app/plugins/privileges/privileges.css]: Indentation

File: app/plugins/privileges/privileges.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    top: 11px;
+    right: 10px;
+    position: absolute;
+    cursor:pointer;
+}
+
+.user-list__element--selected {
+  background-color: #353535;
```
Comment (Autophagy) [inline app/plugins/privileges/privileges.css]: indentation

File: app/plugins/privileges/privileges.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+angular.module('privileges', ['sql'])
+  .factory('UserListService', function(SQLQuery, queryResultToObjects, $q) {
+    var UserListService = {
+      deferred: $q.defer()
+    };
+
+    var stmt = 'select name, superuser from sys.users order by name';
```
Comment (Autophagy) [inline app/plugins/privileges/privileges.js]: Would it be possible to format this (and other sql statments) with capitalisation on keywords? `SELECT name, superuser FROM sys.users ORDER BY name` for example, just to make it a little easier to read

File: app/plugins/privileges/static/i18n/de.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "NAVIGATION": {
+    "PRIVILEGES": "Privileges"
```
Comment (Autophagy) [inline app/plugins/privileges/static/i18n/de.json]: Are these the right german/spanish words? Privilege in german is Privilegien, but im not sure if thats the valid word to use in this context. Worth checking

**Comment by mxm** [comment]
Cool! 😃

**Pull request merged**